src/bundle/Resources/views/themes/admin/content/draft/list.html.twig

```diff
@@ -142,7 +142,7 @@
     <button
         id="confirm-{{ form.remove.vars.id }}"
         type="button"
-        class="btn btn-danger"
+        class="btn ibexa-btn ibexa-btn--primary"
```

Comment:
Not sure if button danger should be changed to primary, but this is probably something that will be clarified later?

Reply:
I believe so.